docker/shadow-hand/kinetic/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM shadowrobot/build-tools:xenial-kinetic-ide
+
+ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/master/ansible/deploy.sh"
+ENV remote_additional_bashrc_cmds="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/F_new_docker_oneliner_kinetic/docker/utils/additional_bashrc_cmds_kinetic"
+
+ENV PROJECTS_WS=/home/user/projects/shadow_robot
+RUN echo "Running one-liner" && \
+    apt-get update && \
+    wget -O /tmp/oneliner "$( echo "$remote_shell_script" | sed 's/#/%23/g' )" && \
+    chmod 755 /tmp/oneliner && \
+    gosu $MY_USERNAME /tmp/oneliner -w $PROJECTS_WS/shadow_robot/base -b "kinetic-devel" -v "kinetic"  && \
+    
+    echo "Updating bash commands" && \
+    wget -O /tmp/additional_bashrc_cmds "$( echo "$remote_additional_bashrc_cmds" | sed 's/#/%23/g' )" && \
+    cat /tmp/additional_bashrc_cmds >> /home/user/.bashrc && \
+
+    echo "Installing mongodb" && \
+    cd $PROJECTS_WS/base_deps/src && \
+    
+    # installing mongocxx driver for the warehouse && \
+    git clone -b 26compat https://github.com/mongodb/mongo-cxx-driver.git && \
+    apt-get update && \
```

Comment:
I get it is not needed since you already has this statement on the top of the Dockerfile